eisbuk-admin/.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "eslint.workingDirectories": [
+        {
+            "directory": "./client",
+            "changeProcessCWD": true
+        }
+
+    ]
+}
```

Comment:
This should be in .gitignore as not everyone is using CS-Code. Maybe add this to the Readme file.